ipa-core/src/protocol/ipa_prf/prf_eval.rs

```diff
@@ -88,7 +88,7 @@ where
         ctx.narrow(&Step::GenRandomMask).prss().generate(record_id);
 
     //compute x+k
-    let mut y = x + AdditiveShare::<Fp25519, N>::expand(k);
+    let mut y = x + AdditiveShare::<Fp25519>::expand(k);
```

Comment:
Maybe change this to `k.expand()`, unless this is in anticipation of something that's needed in the upcoming version that uses additional traits?